assets/scss/_styles_project.scss

```diff
@@ -471,12 +481,20 @@ of .td-content. This applies the same style to any blockquotes that descend from
 Make padding symmetrical (this selector is used in the default styles to apply padding-left: 3rem)
 */
 .pl-md-5, .px-md-5 {
-  padding-right: 3rem;
+  @media (min-width: 768px) {
+    padding-right: 3rem;
+  }
 }
 
 /* Adjust default styles for info banner */
 .pageinfo-primary {
-  max-width: 80%;
+  @media (min-width: 768px) {
+    max-width: 80%;
```

Comment:
No action required, but https://github.com/matrix-org/matrix-spec/issues/761 is related here. The whole thing is a mess.

Reply:
Oh, interesting. Yeah, I had noticed some of the other content having odd widths, too, but tried to keep changes to a minimum here.